server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -199,11 +203,16 @@ public synchronized void init() {
 
   private void initFromLog(File logFile) throws IOException {
     // init the metadata from the operation log
-    mtree = new MTree();
+    mtree = MTree.deserializeFrom(mtreeSnapshotPath);
     if (logFile.exists()) {
       try (FileReader fr = new FileReader(logFile);
           BufferedReader br = new BufferedReader(fr)) {
         String cmd;
+        int idx = 0;
+        while (idx <= mtree.getSnapshotLineNumber()) {
```

Comment:
should this be < ?

Reply:
Well, `return lineNumber++` above affects the logic here. Based on your suggection, it's `<` here.

Reply:
Fixed